example/ck_tile/01_fmha/fmha_fwd.cpp

```diff
@@ -507,90 +496,96 @@ bool run(const ck_tile::ArgParser& arg_parser)
 
     bool pass = true;
 
+    // unify tensor views to [b, h, s, d] layout
+    auto q_host_view = (i_perm ? q_host : q_host.transpose(1, 2));
+    auto k_host_view = (i_perm ? k_host : k_host.transpose(1, 2));
+    auto v_host_view = [&] {
+        auto view = (i_perm ? v_host : v_host.transpose(1, 2));
+        return is_v_rowmajor ? view.transpose(2, 3) : view;
+    }();
+    auto bias_host_view = (i_perm ? bias_host : bias_host.transpose(1, 2));
```

Comment:
Add comment 
// unify bias tensor views to [1, 1, s_q, s_k]  layout
Move it behind of o_host_view